src/hooks/subscription-hooks.js

```diff
@@ -76,6 +77,33 @@ export function useJurorBalancesSubscription(jurorId) {
       movements = [],
     } = jurorData.juror || {}
 
+    const movementsFiltered = movements => {
+      // If the juror is activating from the wallent we need to just show an incoming movement in the Active balance
+      return movements.reduce((previousList, movement, index, originalList) => {
+        const convertedMovement = {
+          ...movement,
+          effectiveTermId: parseInt(movement.effectiveTermId, 10),
+          amount: bigNum(movement.amount),
+        }
+        if (index === 0) {
+          previousList[0] = convertedMovement
+        } else if (
+          ANJMovement[originalList[index - 1].type] ===
+            ANJMovement.Activation &&
+          ANJMovement[movement.type] === ANJMovement.Stake &&
+          originalList[index - 1].createdAt === movement.createdAt
+        ) {
+          previousList[previousList.length - 1] = {
+            ...previousList[previousList.length - 1],
+            type: 'StakeActivation',
+          }
+        } else {
+          previousList[previousList.length] = convertedMovement
```

Comment:
Same here we can do `.push()`
```suggestion
          previousList.push(convertedMovement)
```